geoh5py/workspace/workspace.py

```diff
@@ -669,13 +704,21 @@ def name(self) -> str:
         """
         return self._name
 
+    @property
+    def objects(self) -> dict:
```

Comment:
**question:** why this function when there is already `all_objects()`?

Convenient to have is as a property though.
Should it just return `all_objects()`?